workspaces/keycloak/plugins/catalog-backend-module-keycloak/package.json

```diff
@@ -90,5 +92,15 @@
   "maintainers": [
     "@04kash"
   ],
-  "author": "Red Hat"
+  "author": "Red Hat",
+  "prettier": "@spotify/prettier-config",
```

Comment:
The `prettier` and `lint-staged` config should only be in the workspace root `package.json` file. Please remove this. 👍